flowmachine/flowmachine/core/query.py

```diff
@@ -777,6 +752,14 @@ def invalidate_db_cache(self, name=None, schema=None, cascade=True, drop=True):
         """
         with rlock(self.redis, self.md5):
             con = self.connection.engine
+            try:
+                table_form = self.get_table()
```

Comment:
What does table _form_ mean here?

Reply:
This query, in the form of a Table. Happy to change to something else if you've got a clearer way to express that.

Reply:
Maybe `query_as_table`?

Reply:
(Or even `this_query_as_table`?)